.github/workflows/performance.yml

```
@@ -68,7 +68,7 @@ jobs:
         run: cd kafka_2.13; bin/kafka-server-start.sh config/server.properties &
 
       - name: 'Run kroxy'
-        run: java -jar kroxylicious/target/kroxylicious-1.0-SNAPSHOT.jar --config kroxylicious/example-proxy-config.yml &
+        run: java -jar kroxylicious/target/kroxylicious-*-SNAPSHOT.jar --config kroxylicious/example-proxy-config.yml &
```

keeping the `-SNAPSHOT` does mean we can't run perf tests against the released version. Which seems a little restrictive. 

```suggestion
        run: java -jar kroxylicious/target/kroxylicious-*.jar --config kroxylicious/example-proxy-config.yml &
```

---

Fails on my machine because the pattern catches the sources/javadoc:

```
mvn clean install -Pdist -Dquick
java -jar kroxylicious/target/kroxylicious-*.jar --config kroxylicious/example-proxy-config.yml
Unmatched arguments from index 0: 'kroxylicious/target/kroxylicious-0.2.0-SNAPSHOT-javadoc.jar', 'kroxylicious/target/kroxylicious-0.2.0-SNAPSHOT-sources.jar'
Usage: kroxilicious [-hV] -c=<configFile>
...
 ```

---

MOAR THUNKING NEEDED. Lets merge as is and revisit later.

---

Yeap, that's why I left it with SNAPSHOT. At least we can go on with SNAPSHOT versions